src/components/Loader/pages/PeopleList.tsx

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../../api';
+import { Person } from '../../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from './PersonLink';
+
+export const PeopleList = () => {
+  const [select, setSelect] = useState<string>('');
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  const loadPeople = () => {
+    getPeople()
+      .then(peopleFromServer => setPeople(peopleFromServer))
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code